crates/rolldown/tests/esbuild/default/export_wildcard_fs_node_common_js/artifacts.snap

```diff
@@ -8,18 +8,18 @@ snapshot_kind: text
 
 ```js
 "use strict";
-var ___external = require("./external");
-Object.keys(___external).forEach(function (k) {
+var ___external$1 = require("./external");
+Object.keys(___external$1).forEach(function (k) {
   if (k !== 'default' && !Object.prototype.hasOwnProperty.call(exports, k)) Object.defineProperty(exports, k, {
     enumerable: true,
-    get: function () { return ___external[k]; }
+    get: function () { return ___external$1[k]; }
   });
 });
-var fs = require("fs");
-Object.keys(fs).forEach(function (k) {
+var fs$1 = require("fs");
```

Comment:
unexpected

Reply:
The `LinkingMetadata#require_bindings_for_star_exports` is unless, here removed it.